docs/docsite/rst/community/communication.rst

```diff
@@ -195,48 +119,15 @@ Ansible Community Topics
 
 The `Ansible Community Steering Committee <https://docs.ansible.com/ansible/devel/community/steering/community_steering_committee.html>`_ uses the `community-topics repository <https://github.com/ansible-community/community-topics/issues>`_ to asynchronously discuss with the Community and vote on Community topics in corresponding issues.
 
-Create a new issue in the `repository <https://github.com/ansible-community/community-topics/issues>`_ if you want to discuss an idea that impacts any of the following:
+For more information, see:
 
-* Ansible Community
-* Community collection best practices and `requirements <https://docs.ansible.com/ansible/devel/community/collection_contributors/collection_requirements.html>`_
-* `Community collection inclusion policy <https://github.com/ansible-collections/ansible-inclusion/blob/main/README.md>`_
-* `The Community governance <https://docs.ansible.com/ansible/devel/community/steering/community_steering_committee.html>`_
-* Other proposals of importance that need the Committee or overall Ansible community attention
+* :ref:`creating_community_topic`
+* :ref:`community_topics_workflow`
```

Comment:
@Andersson007 Looks like these two references are undefined on this branch. The docs build fails with:

```
docs/docsite/rst/community/communication.rst:124:0: undefined-label: undefined label: 'creating_community_topic'
docs/docsite/rst/community/communication.rst:125:0: undefined-label: undefined label: 'community_topics_workflow'
```

Reply:
Yep, i saw, https://github.com/ansible/ansible-documentation/pull/1670#issuecomment-2208510351